blt/src/Commands/HumsciCommand.php

```diff
@@ -321,7 +300,24 @@ protected function updateSite($multisite) {
    * @command humsci:keys
    */
   public function humsciKeys() {
-    $this->taskDrush()->drush("rsync @swshumsci.dev:/mnt/gfs/swshumsci.prod/nobackup/apikeys/ @self:../keys")->run();
+    $this->taskDrush()
+      ->drush("rsync @swshumsci.dev:/mnt/gfs/swshumsci.prod/nobackup/apikeys/ @self:../keys")
```

Comment:
The drush alias and the location would be best configured as variables from the BLT yml file. Does this only get run on Acquia's server? If no, I suppose we can ignore making these variables.

Reply:
it would be super quick to change these into blt configuration variables.